src/Domain/CoordinateMaps/Equiangular.cpp

```diff
@@ -0,0 +1,147 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Domain/CoordinateMaps/Equiangular.hpp"
+
+#include <pup.h>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Utilities/DereferenceWrapper.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+
+namespace CoordinateMaps {
+
+Equiangular::Equiangular(const double A, const double B, const double a,
+                         const double b) noexcept
+    : A_(A),
+      B_(B),
+      a_(a),
+      b_(b),
+      length_of_domain_over_m_pi_4_((B - A) / M_PI_4),
+      length_of_range_(b - a),
+      m_pi_4_over_length_of_domain_(1.0 / length_of_domain_over_m_pi_4_),
+      one_over_length_of_range_(1.0 / length_of_range_),
+      linear_jacobian_times_m_pi_4_(length_of_range_ /
+                                    length_of_domain_over_m_pi_4_),
+      linear_inverse_jacobian_over_m_pi_4_(length_of_domain_over_m_pi_4_ /
+                                           length_of_range_) {}
+
+template <typename T>
+std::array<std::decay_t<tt::remove_reference_wrapper_t<T>>, 1> Equiangular::
+operator()(const std::array<T, 1>& xi) const noexcept {
+  return {{0.5 * (a_ + b_ +
+                  length_of_range_ * tan(m_pi_4_over_length_of_domain_ *
+                                         (-B_ - A_ + 2.0 * xi[0])))}};
+}
+
+template <typename T>
+std::array<std::decay_t<tt::remove_reference_wrapper_t<T>>, 1>
+Equiangular::inverse(const std::array<T, 1>& x) const noexcept {
+  return {
+      {0.5 * (A_ + B_ +
+              length_of_domain_over_m_pi_4_ *
+                  atan(one_over_length_of_range_ * (-a_ - b_ + 2.0 * x[0])))}};
+}
+
+template <typename T>
+Tensor<std::decay_t<tt::remove_reference_wrapper_t<T>>,
+       tmpl::integral_list<std::int32_t, 2, 1>,
+       index_list<SpatialIndex<1, UpLo::Up, Frame::NoFrame>,
+                  SpatialIndex<1, UpLo::Lo, Frame::NoFrame>>>
+Equiangular::jacobian(const std::array<T, 1>& xi) const noexcept {
+  return Tensor<std::decay_t<tt::remove_reference_wrapper_t<T>>,
+                tmpl::integral_list<std::int32_t, 2, 1>,
+                index_list<SpatialIndex<1, UpLo::Up, Frame::NoFrame>,
+                           SpatialIndex<1, UpLo::Lo, Frame::NoFrame>>>{
+      linear_jacobian_times_m_pi_4_ *
+      (1.0 +
+       square(tan(m_pi_4_over_length_of_domain_ * (-B_ - A_ + 2.0 * xi[0]))))};
```

Comment:
This can be slightly simplified using trig identities as `1. + square(tan(...))` => `1./square(cos(...))`.  (The same expression occurs again below.)

Reply:
Is it ok if `1.0/square(cos(...))` does an extra division compared to `1.0 + square(tan(...))`? I prefer keeping it as `tan` because the variable change is easier to see that way

Reply:
Discussion during code review:

Factor out tan computation since that's expensive, but otherwise either cos or tan is fine.